pywincffi/kernel32/events.py

```diff
@@ -88,24 +86,25 @@ def OpenEvent(dwDesiredAccess, bInheritHandle, lpName):
         The access desired for the event object.
 
     :param bool bInheritHandle:
-    :param str lpName:
+    :param unicode/str lpName:
+        Type is unicode on Python 2, str on Python 3.
```

Comment:
I'd rephrase so it's clear the doc string is talking about the objects:

```
Type is ``unicode`` on Python 2, ``str`` on Python 3.
```

You could also do:

```
Type is :class:`unicode` on Python 2, :class:`str` on Python 3.
```

I'd suggest the first version though because it render identically in Python 2 and 3 (docs could be built in either version).